src/framework/audio/CMakeLists.txt

```diff
@@ -37,28 +45,25 @@ if (OS_IS_WIN)
         ${CMAKE_CURRENT_LIST_DIR}/internal/platform/win/audiodeviceslistener.cpp
         ${CMAKE_CURRENT_LIST_DIR}/internal/platform/win/audiodeviceslistener.h
     )
-elseif(OS_IS_LIN OR OS_IS_FBSD)
-    if (MUSE_MODULE_AUDIO_JACK)
-        set(DRIVER_SRC
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/jack/jackaudiodriver.cpp
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/jack/jackaudiodriver.h
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/linuxaudiodriver.cpp
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/linuxaudiodriver.h
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/jack/audiodeviceslistener.cpp
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/jack/audiodeviceslistener.h
-        )
-    else()
-        set(DRIVER_SRC
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/linuxaudiodriver.cpp
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/linuxaudiodriver.h
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/audiodeviceslistener.cpp
-            ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/audiodeviceslistener.h
-        )
-    endif()
+elseif(OS_IS_LIN OR OS_IS_FBSD OR MINGW)
+    set(DRIVER_SRC
+        ${CMAKE_CURRENT_LIST_DIR}/internal/audiomidimanager.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/internal/audiomidimanager.h
+        ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/audiodeviceslistener.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/audiodeviceslistener.h
+        ${CMAKE_CURRENT_LIST_DIR}/internal/platform/alsa/alsaaudiodriver.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/internal/platform/alsa/alsaaudiodriver.h
+        ${JACK_SRC}
+       )
 elseif(OS_IS_MAC)
     set(DRIVER_SRC
+        ${CMAKE_CURRENT_LIST_DIR}/internal/audiomidimanager.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/internal/audiomidimanager.h
+        ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/audiodeviceslistener.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/internal/platform/lin/audiodeviceslistener.h
```

Comment:
I think these two lines shouldn't be here

Reply:
it's used by m_devicesListener in AudioMidiManager

Reply:
That's not ideal, since the OSXAudioDriver has its own, more efficient, way of detecting changes in the devices list. Perhaps a way can be found to reuse that?